.gitignore

```diff
@@ -0,0 +1 @@
+storage/
```

Comment:
Could you please add an empty line at the end of this file?

Reply:
Done